src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Logger.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal class Logger
+    {
+        ILogger _logger;
+
+        public Logger() { }
+
+        public Logger(ILogger logger)
+        {
+            _logger = logger;
```

Comment:
The expectation of this constructor is `logger` is not null right? If so, needs null check.